Google Play Music/UpdateDialog.cs

```diff
@@ -80,20 +80,19 @@ public UpdateDialog(string changeLog, string oldVersion, string newVersion)
 
         private void InitializeComponent()
         {
-            // Auto generated by the designer
-            notNowButton = new MaterialRaisedButton();
-            this.updateButton = new MaterialRaisedButton();
-            this.changeLogText = new MaterialLabel();
+            this.notNowButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.updateButton = new MaterialSkin.Controls.MaterialRaisedButton();
+            this.changeLogText = new MaterialSkin.Controls.MaterialLabel();
```

Comment:
I can't find these settings. I attempted to mitigate it using Resharper, but even that looks different than what you had originally. Still learning Git here even though I'm well versed in .Net.